riotfile.py

```diff
@@ -2176,5 +2176,48 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
                 ),
             ],
         ),
+        Venv(
+            name="dogpile_cache",
+            command="pytest {cmdargs} tests/contrib/dogpile_cache",
+            venvs=[
+                Venv(
+                    pys=select_pys(max_version="3.5"),
+                    pkgs={
+                        "dogpile.cache": [
+                            "==0.6.*",
+                            "==0.7.*",
```

Comment:
do we need to test all of these versions? if dogpile is following semver maybe min/max for each Python version range only?

I don't see any version specific code for dogpile in the integration anyways.

Reply:
it's pretty cheap to test so I kept the same versions from tox (only 2 mins)